src/Hl7.Fhir.Support.Poco.Tests/Model/TypedElementToPocoTests.cs

```diff
@@ -116,7 +116,7 @@ public void ParsesResourceWithBackbone()
             };
 
         var poco = toPoco(subject);
-        poco.IsExactly(subject).Should().BeTrue();
+        poco.Should().BeEquivalentTo(subject);
```

Comment:
Be careful undoing this. If we ever make up our mind again, we'll need to call IsExactly again.  We could also make extensions on the Fluent testing library to call IsExactly instead?